calendar.rb

```diff
@@ -0,0 +1,98 @@
+require './event.rb'
+class Calendar
+  def initialize
+    @events = {}
+  end  
+
+  def add_event
+    while true
+      print "Enter the name of Event: "
+      name = gets.chomp
+      if !name.empty?
```

Comment:
when there is only one condition and you want it's negation then use `unless` instead of `if`. for example, here you can do `unless name.empty?`